### PR TITLE
feat: last touches on paywall

### PR DESCRIPTION
**Pull request by kkortes**
Todos:

~Implement mail suggestion "@zootools/email-spell-checker"~ ✅

~Implement checkout.settings.x checks (for phone & name)~ ✅

~Implement (checkout/owner)~ ✅

~Implement native forms~ ✅

~implement "already subscribing" Login-button~ ✅

~implement "continue"-buttons on template === 'BOXES'~ ✅

* Make sure all old properties are supported and working. Missing: `publisher-content-id`

* Make single purchase work as intended #85 

* Make entire paywall disappear if: user is logged in & owns proper entitlement(s)

* When template = "BOXES" the continue buttons should always show and lead to the checkout

* Replace "`document.querySelector('meta[property="sesamy:price"]')`" solution in `SinglePurchase.svelte` with sesamy-js

* e2e-tests: `sesamy-paywall.spec.ts` (test atleast: subscription, single purchase & template = "BOXES").

Post release:
* Enable "template", "background color" and "drop shadow" in Portal > Paywalls https://github.com/sesamyab/portal/pull/1507

### CODE REVIEW

File: packages/demo/src/App.svelte [hunk 1]
```diff
@@ -52,10 +52,10 @@
     </article>
   </section>
   <aside style="min-width: 319px;">
-    <sesamy-paywall
+    <!-- <sesamy-paywall
```
Comment (markusahlstrand) [inline packages/demo/src/App.svelte:55]: Do we want this to be commented out?

**Review by markusahlstrand — APPROVED** [review]
Think this looks good. Should we merge this first or do you want to make the last touches before @frankjonatanschultz ?

File: packages/demo/src/App.svelte (hunk 1 above)
Comment (frankjonatanschultz) [reply packages/demo/src/App.svelte:55]: Yeah its just a second render of the same paywall but in a narrow sidebar, Kim did it to show the responsiveness. I can remove it

**Comment by frankjonatanschultz** [comment]
> Think this looks good. Should we merge this first or do you want to make the last touches before @frankjonatanschultz ?

Yeah lets merge it, i was thinking we keep it and merge fixes in here and cross the todos. But i can copy the todo-list somewhere else 👍

**Pull request merged**